src/templates/extensions/app/react_re/components/Configuration.re

```diff
@@ -97,3 +88,4 @@ let make = (~configuration: Configuration.t) => {
     </ul>
   </div>;
 };
+
```

Comment:
nit: Extra space

Reply:
I think is fixed now with the last two push